driver/normalizer/normalizer.go

```diff
@@ -0,0 +1,122 @@
+package normalizer
+
+import (
+	. "gopkg.in/bblfsh/sdk.v2/uast/transformer"
+	"gopkg.in/bblfsh/sdk.v2/uast"
+)
+
+var Preprocess = Transformers([][]Transformer{
+	{
+		ResponseMetadata{
+			TopLevelIsRootNode: false,
+		},
+	},
+	{Mappings(Preprocessors...)},
+}...)
+
+var Normalize = Transformers([][]Transformer{
+	{Mappings(Normalizers...)},
+}...)
+
+// Preprocessors is a block of AST preprocessing rules rules.
+var Preprocessors = []Mapping{
+	ObjectToNode{
+		OffsetKey: "startOffset",
+		EndOffsetKey: "endOffset",
+	}.Mapping(),
+}
+
+func mapString(key string) Mapping {
+	return MapSemantic(key, uast.String{}, MapObj(
+		Obj{uast.KeyToken: Var("val")},
+		Obj{
+			"Value":  Var("val"),
+			"Format": String(""),
+		},
+	))
+}
+
+func mapIdentifier(key string) Mapping {
+	return MapSemantic(key, uast.Identifier{}, MapObj(
+		Obj{uast.KeyToken: Var("val")},
+		Obj{"Name": Var("val")},
+	))
+}
+
+var Normalizers = []Mapping{
+
+	MapSemantic("function-def-element", uast.FunctionGroup{}, MapObj(
+		Obj{
+			"children": Arr(
+				Obj{
+					uast.KeyType: Var("_type1"),
+					"startOffset": Var("_startOffset1"),
+					"endOffset": Var("_endOffset1"),
+					//uast.KeyPos: Var("_pos1"),
+					"children": Arr(
+						Obj{
+							uast.KeyType: Var("_type3"),
+							uast.KeyToken: Var("name"),
+							"startOffset": Var("_startOffset3"),
+							"endOffset": Var("_endOffset3"),
+							//uast.KeyPos: Var("_pos2"),
+							"children": Var("_children"),
+						},
+					),
+				},
+				Obj{
+					uast.KeyType: Var("_type2"),
+					"startOffset": Var("_startOffset2"),
+					"endOffset": Var("_endOffset2"),
+					//uast.KeyPos: Var("_pos3"),
+					"children": Var("body"),
+				},
+			),
+		},
+		Obj{
+			"Nodes": Arr(
+				UASTType(uast.Alias{}, Obj{
+					"Name": UASTType(uast.Identifier{}, Obj{
+						"Name": Var("name"),
+					}),
+					"Node": UASTType(uast.Function{}, Obj{
+						"Type": UASTType(uast.FunctionType{}, Obj{
+							//"Arguments": Var("args"),
+							"Arguments": nil,
```

Comment:
`IsNil()`?

Reply:
Where is that `IsNil()` defined? I can't find it in the SDK.